lib/grpnice.gi

```diff
@@ -836,6 +836,13 @@ end);
 AttributeMethodByNiceMonomorphism( Size,
     [ IsGroup ] );
 
+#############################################################################
+##
+#M  StructureDescription( <G> )
+##
+AttributeMethodByNiceMonomorphism( StructureDescription,
```

Comment:
@hungaborhorvath : Yes, sometimes the easiest option is to just print the method -- this is `AttributeMethodByNiceMonomorphism`

```
function ( oper, par )
    if 1 <> Length( par )  then
        Error( "need only one argument for ", NameFunction( oper ) );
    fi;
    par := ShallowCopy( par );
    par[1] := par[1] and IsHandledByNiceMonomorphism;
    InstallOtherMethod( oper, "handled by nice monomorphism: Attribute",
     true, par, 0, function ( obj )
          return oper( NiceObject( obj ) );
      end );
    return;
end
```